### ISSUE TITLE
Fine-tuning on my own dataset

### ISSUE THREAD

**Issue by An-Pan** [TimoSaemann/ENet#59]
First I trained ENet on my own dataset (similar with cityscape), and a achieve a good performance on my own test set. But can not get a good result on cityscape dataset just same like the question #14 .

So I try to fine-tuning strategy. I modify the last layer name from "deconv_encoder6_0_0" to "deconv_encoder6_0_0_fine" . And then I start training encoder with "-weights ./ENet/cityscapes_weights_before_bn_merge.caffemodel" 

But when I run test_segmentation.py I can not get any segmentation result at all.

Is there anyone have tried the fine-tuning?
@TimoSaemann

**Comment by ghost** [comment]
Hi @An-Pan , Have you figured out how to fine-tune with this caffe ENet? I was trying to train a 12 classes model, but got nothing as output during inference.. Not sure what the problem is.
